packages/core/src/Stores/client-store.js

```diff
@@ -873,7 +871,6 @@ export default class ClientStore extends BaseStore {
     get country_standpoint() {
         const result = {
             is_united_kingdom: this.is_uk,
```

Comment:
do we need this is_uk flags anymore?

Reply:
I will look into it in a follow-up PR.
https://github.com/binary-com/deriv-app/pull/14475#discussion_r1570044031